### PR TITLE
Improve README file contents

### PR DESCRIPTION
**Pull request by amedina**
* Expanded the contents of the README file. 
* Replaced the idea of having a definition of PWAs in the README by a pointer to Alex Russel's article.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,40 +1,87 @@
...
+
+Details Page #1            |  Details Page #2
+:-------------------------:|:-------------------------:
+![](img/gulliver-details-one.png)  |  ![](img/gulliver-details-two.png)
+
+Gulliver itself has been implemented as a PWA; therefore it is designed to work well in any kind of device, including desktop web browsers (see landing page), and in mobile devices (see details page).
 
 ## Requirements
 
+Gulliver's is built using the [ExpressJS](https://expressjs.com/) web framework for Node.js, and uses the [Google Cloud Platform](https://cloud.google.com/) (GCP) for computing and storage services. Therefore,the following components are required to run the project:
```
Comment (andreban) [inline README.md]: Add space after comma. "Therefore, the following components"...

File: README.md [hunk 2]
```diff
@@ -1,40 +1,87 @@
...
-1. [Google Cloud SDK](https://cloud.google.com/sdk/)
-1. A [Google Cloud Console](https://console.cloud.google.com/) project.
-1. Memcached installed and running on localhost:11211. [Installation instructions](https://cloud.google.com/appengine/docs/flexible/nodejs/caching-application-data).
 
-## Installation
+1. [Google Cloud SDK](https://cloud.google.com/sdk/). A set of tools for GCP that you can use to access the Google Compute Engine and the Google Cloud Storage, which are two components of GCP used by Gulliver.
+
+1. A [Google Cloud Console](https://console.cloud.google.com/) project. A GCC project forms the basis of accessing the GCP. 
+
+1. [Memcached](https://memcached.org/) : a distributed memory object caching system.
```
Comment (andreban) [inline README.md]: We're using a period for Node JS, Google Cloud SDK and the console, and a colon for memcached. Maybe we could standardize on either the period or the colon

File: README.md (hunk 1 above)
Comment (amedina) [inline README.md]: Done.

File: README.md (hunk 2 above)
Comment (amedina) [inline README.md]: Done.

**Pull request merged**